jsonschema2pojo-cli/pom.xml

```diff
@@ -114,14 +114,17 @@
         <dependency>
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter</artifactId>
+            <scope>test</scope>
```

Comment:
Shouldn't the scope come from `dependencyManagement/dependencies` defined in parent `pom` ?

Reply:
Not a good practice IMO: you never know where the dependency is supposed to be used.
I import the junit-bom in parent and that's why the scope should be set now

Reply:
I did a little reading on this.  It does seem like current best practice is to leave scope out of dependency management, unless the scope is `import` or possibly `provided`.  This does live our dependencies in a mixed state, but I don't think it is worth holding up this PR.